index.html

```diff
@@ -6,49 +6,63 @@
     <title>Tour and Travel</title>
     <link rel="stylesheet" href="styles/index.css" />
     <link rel="stylesheet" href="styles/shared.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+<link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Poppins&family=Volkhov&display=swap" rel="stylesheet">
+
   </head>
   <body>
-    <div class="general1">
-      <div class="general2">
-        <div class="text-container">
-          <h5>CATEGORY</h5>
-          <h1>We Offer Best Services</h1>
+    <div class="Homepage-best">
+      <div class="best-services">
+        <div class="Homepage-best-container">
+          <h5 class="Homepage-best-services-1">CATEGORY</h5>
+          <h1 class="Homepage-best-services-2">We Offer Best Services</h1>
         </div>
-        <div class="img-text">
-          <div class="img-text1">
-            <div class="img1">
+        <div class="Homepage-best-services-img-text">
+          <div class="Homepage-best img-text1">
+            <div class="Homepage-best-img1">
               <img src="images/homepage-services-human.png">
+              <span class="Hompage-1"></span>
             </div>
-            <div class="text1">
-              <h4>Guided Tours</h4>
-              <p>sunt qui repellat saepe quo velit aperiam id aliquam placead.</p>
+            <div class="Homepage-best-services-text1">
+              <h4 class="Homepage-best-services">Guided Tours</h4>
+              <p class="Homepage-best-services-3">sunt qui repellat saepe<br> quo velit aperiam id<br> aliquam placead.</p>
             </div>
           </div>
-          <div class="img-text2">
-            <div class="img2">
+          <div class="Homepage-best-services-img-text2">
+            <div class="Hompage-box-shadow"></div>

```

Comment:
`box-shadow: ;` kullanarak yapabilirsin extra bir div'in içine box-shadow verip onu pozisyonlamana gerek yok